readthedocs/builds/migrations/0005_remove-version-alias.py

```diff
@@ -77,7 +79,6 @@ class Migration(migrations.Migration):
             field=readthedocs.builds.version_slug.VersionSlugField(
                 db_index=True,
                 max_length=255,
-                populate_from="verbose_name",
```

Comment:
Is it OK to edit an old migration?

Reply:
yeah, we don't depend on anything here, and it's already applied. We should squash these to remove the old field completely. https://github.com/readthedocs/readthedocs.org/issues/9510

Reply:
👍🏼 on squashing them